src/test/java/com/powsybl/openloadflow/sa/OpenSecurityAnalysisWithActionsTest.java

```diff
@@ -869,7 +870,9 @@ private void testGeneratorAction(boolean dc, LoadFlowParameters.BalanceType bala
         parameters.setDistributedSlack(true).setBalanceType(balanceType);
         parameters.setDc(dc);
         OpenLoadFlowParameters openLoadFlowParameters = new OpenLoadFlowParameters();
-        openLoadFlowParameters.setSlackBusPMaxMismatch(slackBusPMaxMismatch);
+        openLoadFlowParameters.setNewtonRaphsonStoppingCriteriaType(NewtonRaphsonStoppingCriteriaType.PER_EQUATION_TYPE_CRITERIA);
+        openLoadFlowParameters.setMaxActivePowerMismatch(1e-3);
+        openLoadFlowParameters.setSlackBusPMaxMismatch(1e-3);
```

Comment:
So it is less precise than before? I think we have to do a dedicated PR because our unit tests are not homogeneous when we compare LFs and ASs : not same methodology, not same criteria, not same tolerance...

Reply:
With this version, it is as precise as LF tests (and as before) since it is using LoadFlowAssert.DeltaPower to check active power flows differences.

The setting of the "slack bus Pmax mismatch" parameter is here to ensure that we can obtain results precise enough to validate the comparison with LoadFlowAssert.DeltaPower assert.

Maybe I miss something, but I do not see issues to launch tests with different parameters since the asserted values are compared with the same thresholds.

Reply:
It is just for other developers not to "loose" time with these issues.